pkg/integration/replication/suite.go

```diff
@@ -214,17 +281,29 @@ func (suite *baseReplicationTestSuite) WaitForCommittedTx(
 	}
 }
 
-func (suite *baseReplicationTestSuite) SetupCluster(replicas int, syncFollowers int) {
-	suite.StartMaster(syncFollowers)
+func (suite *baseReplicationTestSuite) SetupCluster(syncReplicas, syncAcks, asyncReplicas int) {
+	suite.masterDBName = masterDBName
+
+	suite.StartMaster(syncAcks)
 
 	wg := sync.WaitGroup{}
-	for i := 0; i < replicas; i++ {
+
+	for i := 0; i < syncReplicas; i++ {
```

Comment:
Can't we have one bool parameter inside the suite, and run it in either sync or async mode? I don't see a test scenario where both sync and async need to exist, unless we plan to test those complex topologies?

Reply:
it makes it bit hacky, like for sync scenarios, one has to always imply 0 async, takes away the simplicity from the suite IMO. WDYT?

```
suite.SetupCluster(2, 2, 0)
```